spec.html

```diff
@@ -32156,6 +32156,33 @@ <h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-time-zone-identifiers">
+        <h1>Time Zone Identifiers</h1>
+
+        <p>
+          Time zones in ECMAScript are represented by <dfn variants="time zone identifier">time zone identifiers</dfn>, which must be Strings composed entirely of characters from the Unicode Basic Latin block.
+          Time zone identifiers are compared using ASCII case-insensitive comparisons.
```

Comment:
Is it common knowledge how to do an ASCII case-insensitive comparison, or do we need to spell it out explicitly?

Reply:
You could move the definition in from ECMA-402: https://tc39.es/ecma402/#sec-case-sensitivity-and-case-mapping

Reply:
OK I'll do this.